controllers/users.js

```diff
@@ -26,9 +26,10 @@ usersRouter.post("/", async (req, res, next) => {
 
 })
 
-usersRouter.get("/", async (req, res) => {
-	users = await User.find({})
-	res.json(users.map(u => u.toJSON()))
-})
+// if we don't really need this let's not have it so that no attacker can get the list of users 
```

Comment:
i agree for the final product, but sometimes i use that for testing so I might uncomment at times